### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by nbessi**

### CODE REVIEW

**Comment by nbessi** [comment]
@bwrsandman The fail is silent have you an idea of the fail reasons

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - ODOO="https://github.com/savoirfairelinux/odoo/archive/setuptools-addons.tar.gz"  # Temp until https://github.com/odoo/odoo/issues/185 or https://github.com/odoo/odoo/issues/441 is fixed
+  - git clone https://github.com/OCA/partner-contact-management $HOME/partner-contact-management -b master
+  - git clone https://github.com/OCA/bank-statement-reconcile $HOME/bank-statement-reconcile -b master
+  - git clone https://github.com/OCA/account-financial-tools $HOME/account-financial-tools -b master
```
Comment (bwrsandman) [inline .travis.yml:14]: You're creating 4 different build environments here. Travis makes one build per line in `env`.
The git clones should move down to the `install` section

**Comment by nbessi** [comment]
Thanks :smile: :

**Comment by bwrsandman** [comment]
I think this should wait for https://github.com/OCA/maintainer-quality-tools/pull/29 and use that script instead.

**Comment by nbessi** [comment]
deprecated

**Pull request closed without merging**